doc/build/flashing/configuration.rst

```diff
@@ -0,0 +1,105 @@
+configuration can reside in the following files starting with the highest priority:
+
+ * ``soc.yml`` (in soc folder)
+ * ``board.yml`` (in board folder)
+
+Configuration
+*************
+
+Configuration is applied by using a ``runners`` map with a ``run_once`` child, from here is a
+dictionary of commands as they would be provided to the flash runner e.g. ``--reset`` following by
+a list which specifies the settings for each of these commands (these are grouped by flash runner,
+and by qualifiers/boards). Commands have associated runners that they apply to using a ``runners``
+list value, this can contain ``all`` if it applies to all runners, otherwise must contain each
+runner that it applies to using the runner-specific name. Groups of board targets can be specified
+using the ``groups`` key which has a list of board target sets - board targets are regular
+expression matches, for ``soc.yml`` files each set of board targets must be in a ``qualifiers`` key
+(only regular expression matches for board qualifiers are allowed, the board name must be omitted
+from these entries) and for ``board.yml`` files each set of board targets must be in a ``boards``
+key, these are lists containing the matches which form a singular group. A final parameter ``run``
+can be set to ``first`` which means that the command will be ran once with the first image
+flashing process per set of board targets, or to ``last`` which will be ran once for the final
+image flash per set of board targets.
+
+An example flashing configuration for a ``soc.yml`` is shown below in which the ``--recover``
+command will only be used once for any board targets which used the nRF5340 SoC application or
+network CPU cores, and will only reset the network or application core after all images for the
+respective core have been flashed.
+
+.. code-block:: yaml
+
+  runners:
+    run_once:
+      '--recover':
+        - run: first
+          runners:
+            - nrfjprog
+          groups:
+            - qualifiers:
+                - nrf5340/cpunet
+                - nrf5340/cpuapp
+                - nrf5340/cpuapp/ns
+      '--reset':
+        - run: last
+          runners:
+            - nrfjprog
+            - jlink
+          groups:
+            - qualifiers:
+                - nrf5340/cpunet
+            - qualifiers:
+                - nrf5340/cpuapp
+                - nrf5340/cpuapp/ns
+        # Made up non-real world example to show how to specify different options for different
+        # flash runners
+        - run: first
+          runners:
+            - some_other_runner
+          groups:
+            - qualifiers:
+                - nrf5340/cpunet
+            - qualifiers:
+                - nrf5340/cpuapp
+                - nrf5340/cpuapp/ns
+
+Usage
+*****
+
+Commands that are supported by flash runners can be used as normal, when flashing a sysbuild
+project with multiple images, the flashing configuration run once settings will be applied.
```

Comment:
Sentence to be reviewed: " the flashing configuration run once settings will be applied." I don't understand

Reply:
Reworded